media-gfx/gnofract4d/gnofract4d-4.0.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7} )
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_SETUPTOOLS=no
+
+inherit distutils-r1 xdg
+
+DESCRIPTION="A program for drawing beautiful mathematically-based images known as fractals"
+HOMEPAGE="http://edyoung.github.io/gnofract4d/"
+SRC_URI="https://github.com/edyoung/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+help video"
+
+COMMON_DEPEND="
+	media-libs/libpng:0=
+	virtual/jpeg:0"
+RDEPEND="${COMMON_DEPEND}
+	dev-python/pycairo[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	x11-libs/gtk+:3[introspection]
+	video? ( media-video/ffmpeg[vpx,zlib] )"
+DEPEND="${COMMON_DEPEND}
+	virtual/pkgconfig
+	help? ( app-text/docbook-xsl-stylesheets
+		dev-python/pygobject[${PYTHON_USEDEP}]
+		dev-libs/libxslt
+		x11-libs/gtk+:3[introspection] )"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-xsl.patch
+)
+
+src_prepare() {
+	distutils-r1_src_prepare
+	xdg_environment_reset
```

Comment:
With EAPI-7 this most likely isn't needed, since it does the environment cleaning.

Reply:
Removed.